tests/TestCase/Database/Schema/MysqlSchemaTest.php

```diff
@@ -114,7 +114,7 @@ public static function convertColumnProvider()
             ],
             [
                 'CHAR(25)',
-                ['type' => 'string', 'length' => 25, 'fixed' => true],
+                ['type' => 'char', 'length' => 25, 'fixed' => true],
```

Comment:
Do we need to retain the `fixed` option on `char` types? The type=char should be enough to infer that it is a fixed length type.

Reply:
Should migrations be updated to understand the fixed flag? That might allow us to not have another abstract type.

Reply:
Having fixed flag in test will respond with the same as without flag. I kept it as is.
Because char infers that default value of fixed is true.
I haven't figured the implication on migrations yet with respect to the flag.